ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/akkastreams/dispatcher/Dispatcher.scala

```diff
@@ -38,6 +38,8 @@ trait Dispatcher[Index] {
   ): Source[(Index, T), NotUsed]
 
   def shutdown(): Future[Unit]
+
+  def fail(throwable: Throwable): Future[Unit]
```

Comment:
naming nit: maybe cancel?
On any rate can you please add doc for these two to clarify the distinction?

Reply:
Done